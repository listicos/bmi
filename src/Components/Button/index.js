import React from 'react'
import { Text, StyleSheet, TouchableOpacity } from 'react-native'
import { Colors, Fonts } from '../../Themes'

export default ({ onPress, text }) => (
  <TouchableOpacity style={styles.container} onPress={onPress}>
    <Text style={styles.text}>{text}</Text>
  </TouchableOpacity>
)

const styles = StyleSheet.create({
  container: {
    backgroundColor: Colors.button,
    paddingVertical: 20
  },
  text: {
    ...Fonts.style.h3,
    textAlign: 'center',
    color: Colors.textAccent
  }
})
