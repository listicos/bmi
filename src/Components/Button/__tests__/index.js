import 'react-native'
import React from 'react'
import { shallow } from 'enzyme'
import Button from '../index'

test('Render Button', () => {
  const wrapper = shallow(<Button />)
  expect(wrapper).toMatchSnapshot()
})
