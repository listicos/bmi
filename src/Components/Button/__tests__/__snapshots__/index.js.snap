// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Button 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#0D0E13",
        "borderRadius": 3,
        "flexDirection": "row",
        "height": 45,
        "justifyContent": "center",
        "paddingHorizontal": 25,
      },
      undefined,
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "Raleway",
        "fontSize": 18,
        "fontWeight": "400",
        "letterSpacing": 0.5,
      }
    }
  />
</TouchableOpacity>
`;
